### PR TITLE
Doc modification

### PR DESCRIPTION
**Pull request by adarsh-dell**
# Description
Reordering the subsection of Volume Snapshot feature & steps of installing driver

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| dell/csm#350 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by spriya-m — APPROVED** [review]
Looks good to me.

**Review by shanmydell — CHANGES_REQUESTED** [review]
@adarsh-dell : Do you want to merge it to main or release branch?

**Comment by AkshaySainiDell** [comment]
Looks good.

File: content/docs/csidriver/installation/helm/powerstore.md [hunk 1]
```diff
@@ -117,7 +117,22 @@ Use [v5.0.x](https://github.com/kubernetes-csi/external-snapshotter/tree/v5.0.1/
    - [quay.io/k8scsi/csi-snapshotter:v4.0.x](https://quay.io/repository/k8scsi/csi-snapshotter?tag=v4.0.0&tab=tags)
 - The CSI external-snapshotter sidecar is still installed along with the driver and does not involve any extra configuration.
 
-## Volume Health Monitoring
+#### Installation example 
+
+You can install CRDs and default snapshot controller by running following commands:
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/helm/powerstore.md:122]: Change "following" to "these"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. One minor edit.

File: content/docs/csidriver/installation/helm/powerstore.md (hunk 1 above)
Comment (adarsh-dell) [reply content/docs/csidriver/installation/helm/powerstore.md]: Done.

**Comment by adarsh-dell** [comment]
> @adarsh-dell : Do you want to merge it to main or release branch?

This one is for main branch & there will be another PR for release branch too.

File: content/docs/csidriver/installation/helm/powerstore.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/csidriver/installation/helm/powerstore.md]: @adarsh-dell : could u address the comment here

**Review by shanmydell — APPROVED** [review]

**Pull request merged**